plugins/phrase-conector/src/plugin.tsx

```diff
@@ -30,6 +30,12 @@ registerPlugin(
         type: 'password',
         required: true,
       },
+      {
+        name: 'templateUId',
```

Comment:
why is it called templateUId instead of templateId?

Reply:
using same name as used in Phrase api

Reply:
<img width="690" alt="image" src="https://github.com/user-attachments/assets/15ff9956-f491-4260-b2c6-348c2266183a">